ngrinder-frontend/src/js/components/perftest/detail/RampUp.vue

```diff
@@ -159,118 +181,150 @@
                                 value = value || 0;
                                 return (value / 1000).toFixed(pointCutter);
                             },
+                            count: numTicks,
+                            text: {
+                                position: { y: 10 },
+                            },
+                        },
+                        padding: {
+                            left: 0,
+                            right: 0,
                         },
+                        min: 1,
+                        max: maxX,
                     },
-                    yaxis: {
-                        min: 0,
-                        pad: 10,
-                        max: maxY,
-                        numberTicks: numTicks - 1,
-                        tickOptions: {
-                            show: true,
-                            formatter: (format, value) => {
+                    y: {
+                        tick: {
+                            format: value => {
                                 value = value || 0;
                                 return (value).toFixed(0);
                             },
+                            count: numTicks - 1,
+                            text: {
+                                position: { x: -5 },
+                            },
                         },
+                        padding: {
+                            top: 0,
+                            bottom: 0,
+                        },
+                        min: 0,
+                        max: maxY,
                     },
                 },
-            });
-        }
+                line: { point: false },
+                tooltip: { show: false },
+                legend: { show: false },
+                grid: {
+                    x: { show: true },
+                    y: { show: true },
+                },
+                oninit() {
+                    this.svg.select('g.bb-grid')
+                        .insert('rect', ':first-child')
+                        .attr('width', '100%')
+                        .attr('height', '100%')
+                        .attr('class', 'chart-background');
+                },
+                onrendered(ctx) {
+                    const zoomRect = ctx.$.svg.select('.bb-zoom-rect');
 
-        isPlotTargetExist() {
-            return $('#ramp-up-chart').length !== 0;
+                    ctx.$.svg.select('.chart-background')
+                        .attr('width', +zoomRect.attr('width'))
+                        .attr('height', +zoomRect.attr('height'));
+                },
+            });
         }
     }
 </script>
 
 <style lang="less" scoped>
+    @import '~billboard.js/dist/billboard.min.css';
+    @import '~billboard.js/dist/theme/insight.min.css';
+
     .ramp-up-container {
         width: 460px;
         margin-left: 18px;
 
+        .enable-checkbox {
+            vertical-align: text-bottom;
```

Comment:
Why don't you use bootstrap predefined class?

like this `<td class="align-text-bottom">text-bottom</td>`